bin/alluxio-start.sh

```diff
@@ -415,11 +459,20 @@ main() {
       ;;
   esac
 
-  FORMAT=$1
-  if [[ ! -z "${FORMAT}" && "${FORMAT}" != "-f" ]]; then
-    echo -e "${USAGE}" >&2
-    exit 1
-  fi
+  while getopts "fc:" o; do
+    case "${o}" in
+      f)
+        FORMAT="-f"
+        ;;
+      c)
+        cache="${OPTARG}"
+        ;;
+      *)
+        echo -e "${USAGE}" >&2
+        exit 1
+        ;;
+    esac
+  done
```

Comment:
Could you explain what this change block is intended to do?

Reply:
It's intended just to parse the flags which come after `[MOPT]`. So in this case the old flag `-f` and the new flag + argument `-c [cache]`.